src/components/Menu/MenuItem/__stories__/MenuItem.stories.js

```diff
@@ -47,11 +85,12 @@ export const menuItemOverflowTemplate = args => {
         </Menu>
       </MenuItem>
     </Menu>
-  );
+  ),
+  name: "Overflow"
 };
 
-export const menuItemTooltipTemplate = args => {
-  return (
+export const TooltipStory = {
+  render: () => (
```

Comment:
If the name is not provided we can just return a function

Reply:
Yeah, but again automatic conversion, which I was too lazy too rewrite completely